core/domain/exp_services_test.py

```diff
@@ -62,6 +62,7 @@ class ExplorationServicesUnitTests(test_utils.GenericTestBase):
 
     def setUp(self):
         """Before each individual test, create a dummy exploration."""
+
```

Comment:
Hi @kefeh, this PR is looking pretty good to me. I only have one request: can you remove these blank lines after the docstrings? I'm not sure why they were initially added, and they muddy the [diff](https://github.com/oppia/oppia/pull/2721/files). I think it would be better to follow the other files and not have the extra newline.

Thanks!